.github/workflows/build-release-binaries.yml

```diff
@@ -25,9 +25,11 @@ jobs:
     uses: ./.github/workflows/ios.yml
     secrets: inherit
     with:
-      electrs_endpoint: tcp://blockstream.info:110
-      coordinator_p2p_endpoint: 022ae8dbec1caa4dac93f07f2ebf5ad7a5dd08d375b79f11095e81b065c2155156@34.30.20.202:9045
-      network: mainnet
+      ELECTRS_ENDPOINT: tcp://blockstream.info:110
+      COORDINATOR_PK: 022ae8dbec1caa4dac93f07f2ebf5ad7a5dd08d375b79f11095e81b065c2155156
+      COORDINATOR_HOST: api.10101.finance
```

Comment:
not sure if this will work as we are also using the host for announcing our node. We might have to go with the actual ip address here.

Reply:
I ended up adding the ip addresses to the `nginx.conf` as well, so that we have to change a little as possible.